code/modules/surgery/organs/external/_external_organ.dm

```diff
@@ -267,6 +275,11 @@
 /datum/bodypart_overlay/mutant/snout/get_global_feature_list()
 	return GLOB.snouts_list
 
+/obj/item/organ/external/snout/bunny
+	name = "bunny snout"
+	desc = "So fuzzy!"
```

Comment:
I personally can't imagine someone using this to describe big chungus

Reply:
have you seen a bunny before?

Reply:
Real life bunnies don't walk around on 2 feet so they aren't comparable